Command/RemoveThumbsCommand.php

```diff
@@ -236,4 +263,27 @@ protected function log($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * @return QuestionHelper|DialogHelper
+     */
+    final protected function getQuestionHelper()
```

Comment:
I did not make this `public`. Did you mean `protected`? This should be changed to `private`, or `final protected` is OK?